editor/store/persist.js

```diff
@@ -60,7 +60,8 @@ export function loadAndPersist( store, reducerKey, storageKey, defaults = {} ) {
 		const newStateValue = get( store.getState(), reducerKey );
 		if ( newStateValue !== currentStateValue ) {
 			currentStateValue = newStateValue;
-			window.localStorage.setItem( storageKey, JSON.stringify( currentStateValue ) );
+			const stateToSave = get( reducer( store.getState(), { type: 'REDUX_SERIALIZE' } ), reducerKey );
```

Comment:
Do you think the `REDUX_` prefix adds much value here? It's not as though it's a feature native to Redux, or that we _need_ it to be specific to Redux. I think we could either remove the prefix altogether, or otherwise make it specific to the feature itself (persistence).